core/server/services/members/middleware.js

```diff
@@ -51,7 +51,9 @@ module.exports.use = (siteApp) => {
         }
     });
 
+    // NOTE: this route needs a clear note why (if intentional) is it skipping shared.middlewares.labs.members middleware
```

Comment:
@allouis would love to hear about intention here ;) 

I'm suspecting we want this route to always be available in case some webhooks are coming even though members are disabled?

Reply:
TBH I don't have a reason - I suspect this was just a mistake :shrimp:

Reply:
cool adding the 'skipping shared.middlewares.labs.members' in that case ;)